mne/epochs.py

```diff
@@ -1233,6 +1242,16 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
```

Comment:
I'm not sure I get the point - here, ```epochs``` is the object we create in 1239 and return in 1275. We care about that object's associated event ids.